src/client/eppo-client.ts

```diff
@@ -524,6 +530,35 @@ export default class EppoClient {
     return { variation, action, evaluationDetails };
   }
 
+  /**
+   * For use with 3rd party CMS tooling, such as the Contentful Eppo plugin
```

Comment:
I'd expand this comments a bit more, especially adding more details around each argument. eg.: `flagExperiment` should contain control variation and an array of treatment variations, yadda yadda, plus explain the behavior a bit more around edge cases.

Reply:
I can do that

Reply:
updated!